Add Inverted and Slow functionality to DriveCommands

omentic opened the pull request:
There doesn't seem to be any cameras besides the T265 (@loqoman). Did I just miss it, or is that yet to be merged into master?

Planning on using `.alongWith` for the cameras, unless anyone would rather it be a ParallelCommandGroup within SuperstructureCommands.

dbadb commented on src/main/java/com/spartronics4915/frc2020/commands/DriveCommands.java:
```diff
@@ -32,8 +34,45 @@ public TeleOpCommand(Joystick joy)
+        public void execute()
+        {
+            // Inverted X and Y
+            mDrive.arcadeDrive(-mJoystick.getY(), -mJoystick.getX());
```
i'm not sure if this works... If the normal forward driving, it's my recollection that we need to negate y since push-forward is negative (i think).  If that's true, then shouldn't the y be positive in the reversed case?

dbadb replied:
also: we used to have to specify a dead-band.  Is this no-longer needed?

omentic replied:
A nice feature of the Attack 3 joysticks that Chris uses is a built-in deadband. Will didn't use them in 2018 and we had to specify one.

dbadb commented on src/main/java/com/spartronics4915/frc2020/commands/DriveCommands.java:
```diff
@@ -32,8 +34,45 @@ public TeleOpCommand(Joystick joy)
+        public void execute()
+        {
+            // Don't allow Joystick values to exceed 0.5
+            mDrive.arcadeDrive(mJoystick.getY() * Constants.Drive.kSlowModeMultiplier,
```
again, we might need to negate Y?

omentic replied:
Yes we do, thanks for catching that!

omentic committed:
Create Toggle commands for direction + speed

Those Toggle (and Set) commands are not great and would be much better
implemented as InstantCommands. @dbadb any way to run boolean ^= true in
a passable lambda?

omentic committed:
Implement new Drive commands in RobotContainer
